demos/tests/run_tests.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+
+"""
+Test script for the demos.
+
+The test data directory must contain a subdirectory "ILSVRC2012_img_val"
+containing the ILSVRC2012 dataset.
```

Comment:
Hmm... I think it's a little too verbose for the description of the option itself (and will be even more so if we add other datasets), but it might be a good idea to add it to the overall help message. I'll try that.

Reply:
I ended up using this message as both the docstring and the `argparse` description. I think it looks pretty good.